subsys/periphconf/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 Nordic Semiconductor
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Periphconf integration
+
+config NRF_PERIPHCONF_MIGRATE
+	bool "Generate periphconf_migrated.c"
+	default y
+	depends on \
+	SOC_NRF54H20_CPURAD || \
+	SOC_NRF54H20_CPUAPP
```

Comment:
```suggestion
	depends on SOC_NRF54H20_CPURAD || SOC_NRF54H20_CPUAPP
```
line length is 100

Reply:
no, we use the full 100 line character length

Reply:
Why?